tests/pipelines/test_pipelines_common.py

```diff
@@ -329,7 +329,7 @@ def test_save_load_local(self, expected_max_difference=5e-4):
         inputs = self.get_dummy_inputs(torch_device)
         output_loaded = pipe_loaded(**inputs)[0]
 
-        max_diff = np.abs(to_np(output) - to_np(output_loaded)).max()
+        max_diff = numpy_cosine_similarity_distance(to_np(output).flatten(), to_np(output_loaded).flatten())
```

Comment:
Same here, let's try to use this instead: https://github.com/huggingface/diffusers/pull/4812/files#r1307507791